### PR TITLE
Explain how the data dependent stall comes to be and what it does

### PR DESCRIPTION
**Pull request by jrtc27**

### CODE REVIEW

File: src/performance-methodology/morello-microarchitectural-limitations.md [hunk 1]
```diff
@@ -20,7 +20,9 @@ The following microarchitecture limitations arose due to the constrained timelin
   The underlying Armv8.2-A architecture targeted by the initial Neoverse N1 has
   no concept of data-dependent exceptions on stores, and that
   micro-architecture does not have a structure that supports data-dependent
-  exceptions for stores in an efficient way.
+  exceptions for stores in an efficient way, therefore for capability stores an
```
Comment (rwatson) [inline src/performance-methodology/morello-microarchitectural-limitations.md]: This is a bit of a run-on sentence. Perhaps we should break it into two sentences, with “therefore ...” its own sentence?

File: src/performance-methodology/morello-microarchitectural-limitations.md [hunk 2]
```diff
@@ -20,7 +20,9 @@ The following microarchitecture limitations arose due to the constrained timelin
   The underlying Armv8.2-A architecture targeted by the initial Neoverse N1 has
   no concept of data-dependent exceptions on stores, and that
   micro-architecture does not have a structure that supports data-dependent
-  exceptions for stores in an efficient way.
+  exceptions for stores in an efficient way. Therefore for capability stores an
+  artificial stall is introduced for address translation &mdash; which can
```
Comment (rwatson) [inline src/performance-methodology/morello-microarchitectural-limitations.md:24]: Maybe a mild rephrasing: “For some capability stores, there is therefore an artificial stall introduced ..”?

Comment (jrtc27) [reply src/performance-methodology/morello-microarchitectural-limitations.md:24]: That would beg the question "which capability stores?" to which the answer isn't a function of the nature of the store, but whether the data is already available. I would argue a stall until the data is available covers the case when it's already available as being of zero length.

Comment (jrtc27) [reply src/performance-methodology/morello-microarchitectural-limitations.md:24]: Though I could instead tack on an ", if not already available" at the end of the sentence for clarity?

Comment (rwatson) [reply src/performance-methodology/morello-microarchitectural-limitations.md:24]: Sounds good.

Comment (jrtc27) [reply src/performance-methodology/morello-microarchitectural-limitations.md:24]: Added

**Review by rwatson — APPROVED** [review]

**Pull request merged**